### DISCUSSION TITLE
"An Internet connection is required to install macOS"

### DISCUSSION THREAD

**Discussion by ws1088** [tylernguyen/x1c6-hackintosh#165] [Q&A]
I got to the point where I need to download macOS Monterey, but it said `An Internet connection is required to install macOS`. Is it because I need to pick the right `Kexts` or I need to connect to a docking station with Ethernet? Thanks

**Comment by ws1088** [comment]
I tried the following and i still don't know how to use wifi to connect to internet to download the recovery...

https://github.com/OpenIntelWireless/itlwm/releases (pick [itlwm_xxx.kext]), copy to usb drive (`OC\Kexts`)
https://github.com/OpenIntelWireless/IntelBluetoothFirmware/releases (pick [[IntelBluetoothFirmware-v2.1.0.zip]) copy to usb drive (`OC\Kexts`)
patch `config.plist` with `\patches\Network Patches\Intel.plist` (i modified it to use itlwm instead of the AirportItlwm)

**Comment by FlintSable** [comment]
I ran into a the same issue when trying to install Big Sur and Monterey but the installers just want internet connection these days. I ended up having to get a usb-c thunderbolt to ethernet adapter to get the install to complete. I couldn't get the intel wifi adapter to work so eventually I just got a fenvi BCM94360NG m.2 wifi card and I was able to get wifi working on it without any additional kexts or configs. I don't yet have bluetooth working but i'm still working on that. Did you end up getting the intel wifi card to work?

**Reply by ws1088** [reply]
thanks for the info. i paused on project since 1 month ago. i guess i need to do the same as yours.... thanks!
